src/container/styles.scss

```diff
@@ -96,6 +96,31 @@ the default white background of the container component.
   }
 }
 
+/*
+ * Applies border-radius constraint from parent to children with `overflow:auto`
+ * to prevent content exceeding the parent container if border-radius is not 0.
+ * Stacked containers mirror the parent behavior by having border-radius set to 0.
+ */
+.root.header,
```

Comment:
This is more specific than the `.header` and `.content` selectors above. stylelint enforce the order to be increasing specificity.